mopidy/internal/network.py

```diff
@@ -107,7 +107,8 @@ def __init__(self, host, port, protocol, protocol_kwargs=None,
         self.server_socket = self.create_server_socket(host, port)
         self.address = get_socket_address(host, port)
 
-        self.watcher = self.register_server_socket(self.server_socket.fileno())
+        # GLib wants to get actual socket on win32, so we pass the socket
```

Comment:
Remove this comment. It's enough with the one inside `register_server_socket()`.